.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor
 composer.lock
+.idea/
```

Comment:
Please remove that, it should be in your user gitignore

Reply:
You got it. My bad!